### ISSUE TITLE
Digit rotated

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9909]
**Preferred term label:**
Digit rotated

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Phalanx fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

is this referring to rotation of the fractured digit (or maybe rotation of the fractured finger)?

**Comment by pnrobinson** [comment]
We have been using modifier terms for types of fracture [Fracture type HP:4000042](https://hpo.jax.org/app/browse/term/HP:4000042)

So I added this -- 

Adding new term: 
- ID: HP:6000300
- label: Rotational fracture
- parents: [Fracture type (HP:4000042)](https://hpo.jax.org/app/browse/term/HP:4000042)
- pmid: 
- def: A type of displaced fracture in which there has been a rotation of the distal fracture fragment in relation to the proximal portion.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9909
- synonyms:

**Comment by MickeySegal** [comment]
We are not trying to characterize the fracture in the finding in this NTR.  We are trying to point the clinician towards observing the physical finding of a rotated digit.  So I created a new finding for the radiology that you described here and used it, but it would be good to also have a finding for the finger rotation.
